Added 'Strings added in Underscore templates' section to 'Translate theme strings' page (#6926)
magento/devdocs: src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md

__($this->config->getData('label'))
```

## Strings added in Underscore templates {#add_strings_underscore_html}

To ensure that the text you add in `.html` Underscore templates is collected by the `i18n` tool and added to the dictionary, use the `_.i18n('')` Underscore function.

-  When only a string is added in template use the following example:

   ```html
   <%= _.i18n('Hello') %>
   ```

> -  When a string contains a variable, to add a placeholder for this variable to the string stored in the dictionary, use the syntax similar to the following:

Reviewer: Please, simplify this statement

```suggestion
-  When the string contains a variable, use the variable placeholder for processing string stored in the dictionary: 
```

Author: Thank you for suggestion, the changes has been added.
